packages/mdl-radio/event-bus.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Class to coordinate between between components without strict dependencies.
+ */
+export default class EventBus {
```

Comment:
Ooh, I take this back. Didn't realize the native checked property will in fact automatically propagate to all corresponding input elements. So we can do just as you mentioned. Thanks!